### ISSUE TITLE
[New first run onboarding] [Accessibility]  - ToS card is incorrectly displayed with dynamic text and cannot be agreed

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#28737]
## Steps to reproduce
1. Install FF and open the app.
1. Set dynamic text to a larger size from FF settings and reopen FF.

**Actual Result:**

ToS card is incorrectly displayed.

Its content can't be visualized entirely, it can’t be scrolled, it is basically frozen.

“Continue” button is not visible.

#### **Expected Result:**
The ToS card can be scrolled for the entire content to be visible.

The card becomes scrollable after force close/put in background and reopened.

### Device & build information
* Device: iPhone 15 (17.6)
* Build version: v9000 (59991)

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXIOS-13207)

**Comment by data-sync-user** [comment]
➤ Miruna Curtean commented:

Verified fixed on v9000 (62414) with iPhone 14 Pro Max (26), iPad Mini A17 Pro (iOS 26) and iOS 11 (18.7).

!RecordIt-FE982052-9B4E-47BD-B677-98F761E842E2.mp4|width=396,alt="RecordIt-FE982052-9B4E-47BD-B677-98F761E842E2.mp4"!
